.svgo.yml

```diff
@@ -0,0 +1,5 @@
+plugins:
+  removeViewBox: false
+  addAttributesToSVGElement:
+    attributes:
+      focusable: false
```

Comment:
@austin94 please check your commits – this yaml doesn't do anything. Ensure that you see `viewBox="0 0 12 12"` and `focusable="false"` are on the SVG that's being produced in source. The following syntax (pulled from `svg-icons`) does exactly that:

```yaml
plugins:
  - addAttributesToSVGElement:
      attributes:
        - focusable: false
  - removeViewBox: false
```

Reply:
This is now corrected. Not sure why it was still referencing the older config.